bridge/discord.go

```diff
@@ -453,6 +457,14 @@ func pmTargetFromContent(content string) (nick, newContent string) {
 	nick = subs[0]
 	newContent = strings.TrimPrefix(subs[1], " ")
 
+	nickParts := strings.Split(nick, "@")
+
+	if nickParts[1] != discriminator {
+		return "", ""
+	}
+
+	nick = nickParts[0]
```

Comment:
Will crash if the nick is not valid

Reply:
Didn't think of that, I'll fix up the code to use a check on the length of nickParts. Or are you saying the strings.Split call will cause the crash?

Reply:
Added a  fix.